### PR TITLE
Add composite action path to GITHUB PATH

### PR DESCRIPTION
**Pull request by zmiklank**
Run python script with python interpreter directly

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -134,15 +134,17 @@ runs:
         username: ${{ inputs.registry_username }}
         password: ${{ inputs.registry_token }}
 
+    - run: echo "${{ github.action_path }}" >> "$GITHUB_PATH"
```
Comment (phracek) [inline action.yml]: Documentation would be welcome. and I have missed the content. Why this is needed?

**Review by phracek — CHANGES_REQUESTED** [review]
Where is the definition of which python interpreter is used? In case we use python-3.11 and support it and test it, then use @setup-python GitHub Action with py311 environment.

File: action.yml (hunk 1 above)
Comment (zmiklank) [reply action.yml]: I have added some description to the action and to the commit message.

**Comment by zmiklank** [comment]
> Where is the definition of which python interpreter is used? In case we use python-3.11 and support it and test it, then use @setup-python GitHub Action with py311 environment.

It is the default python available on the test environment the user chooses. In our case it is the python from ubuntu 20.04.
I think that this approach should be sufficient for this use case.

**Review by phracek — APPROVED** [review]

**Pull request merged**